plugins/modules/k8s.py

```diff
@@ -424,19 +399,20 @@ def main():
         ("template", "src"),
         ("name", "generate_name"),
     ]
-    module = AnsibleModule(
+
+    from ansible_collections.kubernetes.core.plugins.module_utils.k8s.core import (
+        AnsibleK8SModule,
+    )
+    from ansible_collections.kubernetes.core.plugins.module_utils.k8s.runner import (
```

Comment:
```
ModuleNotFoundError: No module named 'ansible_collections.kubernetes.core.plugins.module_utils.k8s.runner
```

Reply:
This PR and the other migrating k8s_info required the runner to be merged first.

Reply:
Yes, I saw that PR later. My bad.